### DISCUSSION TITLE
PyOxy 0.1

### DISCUSSION THREAD

**Discussion by indygreg** [indygreg/PyOxidizer#560] [General]
Initial release.

<hr /><em>This discussion was created from the release <a href='https://github.com/indygreg/PyOxidizer/releases/tag/pyoxy/0.1.0'>PyOxy 0.1</a>.</em>

**Comment by cipriancraciun** [comment]
Nice job!  I was using your custom static Python builds to have a stable run-time environment, but now with `pyoxy` that is even simpler!  One file and that's it!  (I'm curious how well it does with non pure-Python libraries.)

Projects like yours are what keeps me hoping Python doesn't devolve into the madness of continuously breaking run-time environments.

Thanks again, keep up the good work!
